RESPONSIBILITIES.md

```diff
@@ -38,7 +38,7 @@ Maintainers are active and visible members of the community, and have [maintain-
 
 ### Uphold Code of Conduct
 
-Model the behavior set forward by the [Code of Conduct](CODE_OF_CONDUCT.md) and raise any violations to other maintainers and admins.
+Model the behavior set forward by the [Code of Conduct](CODE_OF_CONDUCT.md) and raise any violations to other maintainers and [members of the admin team](https://github.com/orgs/opensearch-project/teams/admin).
```

Comment:
_From https://opensearch.slack.com/archives/C051JEH8MNU/p1686591138909899_

This link isn't visible to the public, looking at a blog post on teams [1], `People who are not members of the organization cannot view any teams.`  Seems like this change doesn't expose this information publicly.

I think we should drop the url since it won't work for anyone that isn't a member of the project, what do you think about coming up with a page like maintainers but for admins that links the team, but also has the list of admins available? 

- [1] https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility

Reply:
How do you think about pointing to https://github.com/opensearch-project/.github/blob/main/ADMINS.md?